pulsar-broker/src/test/java/org/apache/pulsar/broker/stats/PrometheusMetricsTest.java

```diff
@@ -310,11 +310,11 @@ public void testPerTopicStats() throws Exception {
         assertEquals(cm.get(1).tags.get("namespace"), "my-property/use/my-ns");
 
         cm = (List<Metric>) metrics.get("pulsar_producers_count");
-        assertEquals(cm.size(), 3);
-        assertEquals(cm.get(1).tags.get("topic"), "persistent://my-property/use/my-ns/my-topic2");
+        assertEquals(cm.size(), 2);
```

Comment:
Well, if you're already here and adding new functionality (broker-level metrics), I would advise adding a test to ensure it works and will never break. Probably in a new method.

Reply:
> Well, if you're already here and adding new functionality (broker-level metrics), I would advise adding a test to ensure it works and will never break. Probably in a new method.

The broker-level metrics unit test has been added at the method ```testPerBrokerStats```.

commit:
https://github.com/apache/pulsar/pull/19047/commits/e404afd78cd965c27fe7247352d377713880cf9b